dump/Module/module5/modul7.html

```diff
@@ -0,0 +1,100 @@
+                </inhalte>
+                <lernergebnisse>
+                    Studierende haben nach erfolgreichem Abschluss dieses Moduls ein vertieftes Verständnis aktueller Methoden und typischer Anwendungen im Bereich Data Science. Teilnehmer können wissenschaftliche Veröffentlichungen systematisch aufarbeiten, und deren Inhalte strukturiert zusammenfassen und kritisch bewerten.
+                </lernergebnisse>
+                <literatur>
+                    Zu dieser Veranstaltung gibt es kein Lehrbuch. Einige relevante Grundlagen finden sich in:
+
+                    * R for Data Science (by Garrett Grolemund and Hadley Wickham) O’Reilly Media
+                    * Statistics in a Nutshell, 2nd Edition, A Desktop Quick Reference, Sarah Boslaugh, O’Reilly Media
+                    * Doing Data Science –Straight Talk from the Frontline, Cathy O’Neil, Rachel Schutt, O’Reilly Media
+                    * Data Analytics with Hadoop, Benjamin Bengfort & Jenny Kim, O‘Reilly Media
+                    * Deep Learning by Ian Goodfellow and Yoshua Bengio and Aaron Courville http://www.deeplearn-ingbook.org/
+                </literatur>
+                <verwendbarkeit>
+                    * Wahlpflichtbereich Praktische oder Technische Informatik
+                    * Schwerpunktbereich
+                    * Individuelle Ergänzung
+                    * Anwendungsfach für den Ergänzungsbereich im Master-Studiengang Mathematik
+                    * Wahlbereich im Master-Studiengang Artificial Intelligence and Data Science
+                </verwendbarkeit>
+                <voraussetzungenBestehen>
+                    * Schriftliche Ausarbeitung zum gewählten Thema
+                    * Begutachtung von anderen Ausarbeitungen
+                    * Präsentation des eigenen Themas
+                    * Aktive Teilnahme an Diskussionen
+                </voraussetzungenBestehen>
+                <haeufigkeit>
+                    unregelmäßig
+                </haeufigkeit>
+                <sprache>
+                    Englisch
+                </sprache>
+            </beschreibung>
+            <voraussetzungenTeilnahme>
+                Bachelor-Studierende müssen folgende Module erfolgreich abgeschlossen haben:
+
+                * „Programmierung”
+                * „Rechnerarchitektur“
+                * „Algorithmen und Datenstrukturen”
+                * „Theoretische Informatik”
+            </voraussetzungenTeilnahme>
+            <semesters>
+                <semester>
+
+                </semester>
+            </semesters>
+            <zusatzfelder>
+                <zusatzfeld>
```

Comment:
So erstellt der doch ein leeres Zusatzfeld, oder?

Reply:
haben das problem in mops_fillDatabaseWithModuls beim parsen gelöst, Zusatzfelder mit leerem Inhalt werden ignoriert